.github/workflows/update.yml

```diff
@@ -12,6 +12,10 @@ jobs:
       - uses: actions/setup-node@v2
       - run: npm ci
       - run: npm install --save-exact @octokit/webhooks-definitions@latest
+      - uses: gr2m/await-npm-package-version-action@v1
```

Comment:
Shouldn't this be *before* we install the package, if its meant to making us wait for the package to be found?

Reply:
I believe so

Reply:
Oopsies sorry I should have caught that myself. Great catch @G-Rath